### DISCUSSION TITLE
对“内存优化”功能进行的一些讨论

### DISCUSSION THREAD

**Discussion by RimacC2-EV** [Meloong-Git/PCL#3828] [讨论]
咱们PCL的“内存优化”是什么原理？今天拿我16GB的HDD电脑试了一下然后就卡爆了······差不多得等上3-4分钟才恢复好（不是喷PCL······）
或者有知情人士知道它和 Smart RAM 里的 “智能清理”  “深度清理”有什么区别吗······
![image](https://github.com/Hex-Dragon/PCL2/assets/125170682/c6c279d0-f13e-4f22-876e-ea0057cea006)

**Comment by WForst-Breeze** [comment]
#2090 #2500

**Comment by RimacC2-EV** [comment]
感谢！


话说各位PCL爱好者都这么会秒回吗

**Reply by MoYuan-CN** [reply]
我们（至少我）订阅了 PCL 整个仓库的新内容事件提醒
再加上还有 #1930 等文档的存在，回复速度通常都很快的
xD！

**Reply by WForst-Breeze** [reply]
大部分活跃的用户基本都秒回，我之前也挺活跃，但现在因为个人原因相比之前差多了哈哈哈，已经有新一批爱好者接力了

**Reply by RimacC2-EV** [reply]
太好力（乱入）
我也去订个阅 👍 :D
